lib/TH/THMemoryFile.c

```diff
@@ -177,7 +177,7 @@ static int THMemoryFile_mode(const char *mode, int *isReadable, int *isWritable)
       size_t i;                                                           \
       for(i = 0; i < n; i++)                                            \
       {                                                                 \
-        ssize_t nByteWritten;                                           \
+        int nByteWritten;                                               \
```

Comment:
I was asking because on my x86_64 ubuntu, `ssize_t` is actually a `long`.
I am really not sure what should be done here since this variable can be assigned either an `int` or a `size_t`.